java/dataset/src/main/java/org/apache/arrow/dataset/substrait/util/ConvertUtil.java

```diff
@@ -24,10 +24,12 @@
 public class ConvertUtil {
 
   /**
-   * Convert map to substrait Expression.
+   * Convert map to substrait Expression Literal.
+   * @param values config map
    *
-   * @return Substrait Expression
+   * @return substrait Expression Literal
```

Comment:
nit:
```suggestion
   * @return Substrait Expression Literal
```

Reply:
I find the brief comments starts with uppercase, but param and returns starts with lower case, which is the expected syntax? https://github.com/apache/arrow/blob/main/java/dataset/src/main/java/org/apache/arrow/dataset/jni/JniWrapper.java#L80

Reply:
Oh sorry, you're right.

Reply:
"Substrait" is a proper noun, though